@mattclay The ones which where reported by anibot last time:
```
lib/ansible/modules/remote_management/foreman/katello.py:0:0: E309 version_added for new option (action) should be 2.7. Currently 0.0
lib/ansible/modules/remote_management/foreman/katello.py:0:0: E309 version_added for new option (verify_ssl) should be 2.7. Currently 0.0
lib/ansible/modules/remote_management/foreman/katello.py:0:0: E328 Choices value from the documentation ('name') is not compatible with type 'dict' defined in the argument_spec
test/sanity/validate-modules/ignore.txt:1089:1: A102 Remove since "lib/ansible/modules/remote_management/foreman/katello.py" passes "E322" test
test/sanity/validate-modules/ignore.txt:1090:1: A102 Remove since "lib/ansible/modules/remote_management/foreman/katello.py" passes "E325" test 
```

So in my opinion these are fine as the structure of the module does not allow to document it correctly with the choices method, as "params" is a dict, so ive written all choices in the comment of that parameter. Also the first 2 Issues are Ok because verify_ssl was undocumented but i dont know why action is poping up :) .

--- a/lib/ansible/modules/remote_management/foreman/katello.py
+++ b/lib/ansible/modules/remote_management/foreman/katello.py
@@ -86,21 +86,6 @@ options:
             - "prior ([lifecycle_environment,None])"
             - "content_view ([activation_key,None])"
             - "lifecycle_environment ([activation_key,None])"
-        choices:
-            - name
-            - organization
-            - content
-            - product
-            - basearch
-            - releaserver
-            - sync_date
-            - interval
-            - repositories
-            - from_environment
-            - to_environment
-            - prior
-            - content_view
-            - lifecycle_environment
         required: true
     task_timeout:
         description:

--- a/test/sanity/validate-modules/ignore.txt
+++ b/test/sanity/validate-modules/ignore.txt
@@ -1081,8 +1081,6 @@ lib/ansible/modules/packaging/os/yum_repository.py E324
 lib/ansible/modules/packaging/os/zypper.py E326
 lib/ansible/modules/remote_management/foreman/foreman.py E322
 lib/ansible/modules/remote_management/foreman/foreman.py E325
-lib/ansible/modules/remote_management/foreman/katello.py E322
-lib/ansible/modules/remote_management/foreman/katello.py E325
 lib/ansible/modules/remote_management/hpilo/hpilo_boot.py E324
 lib/ansible/modules/remote_management/hpilo/hpilo_boot.py E326
 lib/ansible/modules/remote_management/ipmi/ipmi_boot.py E326

